src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/HearingsHelper.java

```diff
@@ -106,12 +111,32 @@ public static void updatePostponedDate(CaseData caseData) {
                 if (HEARING_STATUS_HEARD.equals(dateListedType.getHearingStatus())) {
                     checkStartFinishTimes(errors, dateListedType,
                             hearingTypeItem.getValue().getHearingNumber());
+                    checkIfDateInFuture(errors, dateListedType);
                 }
             }
         }
         return errors;
     }
 
+    private static void checkIfDateInFuture(List<String> errors, DateListedType dateListedType) {
+        if (isDateInFuture(dateListedType.getHearingTimingStart())) {
+            errors.add(HEARING_START_FUTURE);
+        }
+        if (isDateInFuture(dateListedType.getHearingTimingResume())) {
+            errors.add(HEARING_RESUME_FUTURE);
+        }
+        if (isDateInFuture(dateListedType.getHearingTimingBreak())) {
+            errors.add(HEARING_BREAK_FUTURE);
+        }
+        if (isDateInFuture(dateListedType.getHearingTimingFinish())) {
+            errors.add(HEARING_FINISH_FUTURE);
+        }
+    }
+
+    private static boolean isDateInFuture(String date) {
+        return !Strings.isNullOrEmpty(date) && LocalDateTime.parse(date).isAfter(LocalDateTime.now());
```

Comment:
If the date is empty, will it return false?

Reply:
Yes it will return false so it will skip the validation but in the case of start and finish, they are mandatory fields so they won't be empty. WIth resume and break, they are optional hence why it is needed

Reply:
Harpreet's bound to have written a unit test for this scenario ;o)

Reply:
Created :)